starlette/responses.py

```diff
@@ -14,6 +14,11 @@
 from starlette.datastructures import URL, MutableHeaders
 from starlette.types import Receive, Scope, Send
 
+# Workaround for adding samesite support to pre 3.8 python
+from http.cookies import Morsel
+
+Morsel._reserved[str("samesite")] = str("SameSite")  # type: ignore
```

Comment:
We shouldn't need the `str()` wrappers here.

(Eg. the comments against https://stackoverflow.com/questions/50813091/how-do-i-set-the-samesite-attribute-of-http-cookies-in-python?noredirect=1#comment93090548_50813092 indicate that they were only there for Python 2 support)

Reply:
Yeah, you're right.